easybuild/easyconfigs/f/freebayes/freebayes-0.9.21-goolf-1.7.20.eb

```diff
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'freebayes'
+version = '0.9.21'
+
+homepage = 'https://github.com/ekg/freebayes'
+description = """ Bayesian haplotype-based polymorphism discovery and genotyping. """
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'} 
+
+# no direct download url. As described in documentation the download has been done with
+# $> git clone --recursive git://github.com/ekg/freebayes.git
+# Then compressed and placed in the sources folder with name freebayes-0.9.21.tar.bz2
+sources = [SOURCELOWER_TAR_BZ2]
+
+builddependencies = [('CMake', '2.8.12', '', ('GCC', '4.8.4'))]
+
+parallel = 1
+
+# workaround to avoid the error "The C compiler identification is unknown"
```

Comment:
to be honest I don't remember why I had to do this. It was a quick workaround so the build works. I need to review it with more detail....